lms/static/js/Markdown.Converter.js

```diff
@@ -1285,27 +1285,12 @@ else
         //  attacklab: Utility functions
         //
 
-        var _problemUrlChars = /(?:["'*()[\]:]|~D)/g;

```

Comment:
Why did you decide to delete this method (are we not concerned with some of these "problem" chars)?